cmd/kube-apiserver/app/server.go

```diff
@@ -566,11 +566,12 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 		return nil, fmt.Errorf("error in initializing storage factory: %s", err)
 	}
 
-	// keep Deployments, NetworkPolicies, Daemonsets and ReplicaSets in extensions for backwards compatibility, we'll have to migrate at some point, eventually
+	storageFactory.AddCohabitatingResources(networking.Resource("networkpolicies"), extensions.Resource("networkpolicies"))
```

Comment:
Do we want a comment to remove this during v1.9 devel?

Reply:
Along with the extensions type completely?

Reply:
we shouldn't remove this until the extensions type is removed and migration is enforced. should networkpolicies be added to `update-storage-objects.sh`?